src/plugins/foxPage/components/FoxOpportunity.tsx

```diff
@@ -0,0 +1,71 @@
+import { Box, Flex } from '@chakra-ui/layout'
+import { Button, Text as CText } from '@chakra-ui/react'
+import { useTranslate } from 'react-polyglot'
+import { fox } from 'test/mocks/assets'
+import { Amount } from 'components/Amount/Amount'
+import { AssetIcon } from 'components/AssetIcon'
+import { Card } from 'components/Card/Card'
+import { Text } from 'components/Text/Text'
+
+type MainOpportunityProps = {
+  apy: string
+  assetSymbol: string
+  tvl: string
+  balance: string
+  onClick: () => void
+}
+
+export const FoxOpportunity = ({
+  apy,
+  tvl,
+  assetSymbol,
+  balance,
+  onClick,
+}: MainOpportunityProps) => {
```

Comment:
```suggestion
export const FoxOpportunity: React.FC<MainOpportunityProps> = ({
  apy,
  tvl,
  assetSymbol,
  balance,
  onClick,
}) => {
```

non-blocking comment, but this is always the nicer way to do things - type the function, not the args